flink-core/src/main/java/org/apache/flink/core/memory/MemorySegmentFactory.java

```diff
@@ -130,7 +130,7 @@ public static MemorySegment allocateUnpooledOffHeapMemory(int size) {
      */
     public static MemorySegment allocateUnpooledOffHeapMemory(int size, Object owner) {
```

Comment:
Should we rename the factory methods to reflect which type of off heap memory they allocate?

Reply:
Addressed in later commit.